e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/E2ERunnerConfig.scala

```diff
@@ -62,7 +62,7 @@ case class E2ERunnerConfig(rawFormat: String = "parquet",
     rawFormat.toLowerCase() match {
       case "parquet" => "--raw-format parquet"
       case "json" => "--raw-format json"
-      case "csv" => s"--raw-format csv --delimiter $csvDelimiter --header $csvHeader"
+      case "csv" => s"--raw-format csv --delimiter ${'"'}$csvDelimiter${'"'} --header $csvHeader"
```

Comment:
```suggestion
      case "csv" => s"""--raw-format csv --delimiter "$csvDelimiter" --header $csvHeader"""
```

I think this is called a smart string. `""" something """` and then you can use whatever chars in between them. Look at this [video](https://www.youtube.com/watch?v=RWWotQXltkE)

Reply:
👍 agree, the smart string is nicer solution